newrelic-integration-e2e/internal/executor/executor.go

```diff
@@ -1,22 +1,59 @@
 package executor
 
 import (
+	"os/exec"
+	"time"
+
 	"github.com/newrelic/newrelic-integration-e2e/pkg/agent"
 	"github.com/newrelic/newrelic-integration-e2e/pkg/settings"
+	"github.com/sirupsen/logrus"
 )
 
-func Exec(ag agent.Agent, settings settings.Settings) error{
+func Exec(ag agent.Agent, settings settings.Settings) error {
 	spec := settings.Spec()
 	for i := range spec.Scenarios {
 		scenario := spec.Scenarios[i]
 		settings.Logger().Debugf("[scenario]: %s", scenario.Description)
-		if err:=ag.SetUp(scenario);err!=nil{
+		if err := ag.SetUp(settings.Logger(), scenario); err != nil {
```

Comment:
why inline redefining each time err? I personally find it more difficult to read. However I guess many people would disagree with me :)

Reply:
I agree with those people hahahah